src/sections/upload-dataset-files/uploaded-files-list/add-tags-modal/AddTagsModal.module.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+
+.add_tags_form {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+}
+
+.tags {
+    display: flex;
+}
+
+.tags_select{
+    width: 100%;
+}
+
+.tag_options {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+}
+
+.tag_info {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    color: $dv-subtext-color;
+}
+
+.apply_button {
+    height: 2.4rem;
+}
```

Comment:
Same end of line here and css auto formatting not applying?

Reply:
My auto formatting removes the empty last line... Added empty line here now.